macros/REST_Axion_XMMAngleEffCSV.C

```diff
@@ -0,0 +1,126 @@
+#include "TCanvas.h"
+#include "TLatex.h"
+#include "TLegend.h"
+#include "TRestAxionWolterOptics.h"
+#include "TRestTask.h"
+#include "TRestTools.h"
+#include "rapidcsv.h"
+
+#ifndef RestTask_Axion_XMMAngleEff
+#define RestTask_Axion_XMMAngleEff
+
+//*******************************************************************************************************
+//*** Description: It creates an efficiency curve from different runs under different angles. 
+//*** It's also possible to compare data from a csv file with this curve.
+//*** 
```

Comment:
It would be better to provide the CSV filename as an argument. Also the documentation could say what are the contents of the CSV file.

Reply:
CSV files could be placed at the `optics` directory inside https://github.com/rest-for-physics/axionlib-data/tree/master/optics